integration-tests/tests.ts

```diff
@@ -750,7 +751,8 @@ test("feature_flag_in_function", async ({ page }) => {
     await awaitAnalysis(page, timestamp);
 
     await page.waitForSelector(".return-value");
-    const expectedText = "but + only works on Ints.";
+
+    const expectedText = "Try using Float::+, or use Float::truncate to truncate Floats to Ints.";
```

Comment:
TODO: this will pass in F# but not in OCaml. fix that.

Reply:
You could pass it if it includes either the OCaml OR the F# error message.

Reply:
We must have fixed this error message since I did this - the error message is consistent for me now, it seems.